### PR TITLE
fix: cleanup primevue stylesheet

### PR DESCRIPTION
**Pull request by J0taFerreira**
cleaning up primevue stylesheet:
- removed FAM specific style change
- some auto formatting

### CODE REVIEW

File: style-sheets/primevue-components-overrides.scss [hunk 1]
```diff
@@ -391,7 +392,57 @@ label {
 .#{variables.$primevue-prefix}-card .#{variables.$primevue-prefix}-card-footer {
     padding: 0;
 }
+
 /* ------------------------------------ primevue table --------------------------------- */
+
+.custom-access-table {
```
Comment (MCatherine1994) [inline style-sheets/primevue-components-overrides.scss:398]: Hi Jota, we don't put any custom class override in this file. These style change actually apply to the primevue data table component, and the class name for primevue data table component is ".p-data-table". 
Screen Shot 2023-08-24 at 11 39 54 AM

File: style-sheets/primevue-components-overrides.scss [hunk 2]
```diff
@@ -391,7 +392,57 @@ label {
...
+    background: transparent;
+    border-radius: 0.25rem 0.25rem 0 0;
+    border: 0.125rem solid map.get(lightTheme.$light-theme, 'border-subtle-00');
+}
+
+.custom-table-header {
```
Comment (MCatherine1994) [inline style-sheets/primevue-components-overrides.scss:404]: This table header is what we added for FAM, primevue data table component doesn't have a header, so we shouldn't put it here. Let's put it back to our FAM repo.

File: style-sheets/primevue-components-overrides.scss [hunk 3]
```diff
@@ -391,7 +392,57 @@ label {
...
+
+    span {
+        @extend %body-compact-01;
+        margin: 0;
+        padding: 0;
+        color: map.get(lightTheme.$light-theme, 'text-secondary');
+    }
+}
+
+.span-icon {
```
Comment (MCatherine1994) [inline style-sheets/primevue-components-overrides.scss:420]: also this one I'm not sure where we use it, but we only put primevue component style change in this file.

File: style-sheets/primevue-components-overrides.scss [hunk 4]
```diff
@@ -391,7 +392,57 @@ label {
...
+    margin-left: 0.9375rem;
+}
+
+.remove-action {
```
Comment (MCatherine1994) [inline style-sheets/primevue-components-overrides.scss:424]: please check this one as well

File: style-sheets/primevue-components-overrides.scss [hunk 5]
```diff
@@ -391,7 +392,57 @@ label {
...
+    color: map.get(lightTheme.$light-theme, 'text-error');
+}
+
+.#{variables.$primevue-prefix}-datatable
+.#{variables.$primevue-prefix}-sortable-column
+.#{variables.$primevue-prefix}-sortable-column-icon {
+    display: none;
+}
+
+.#{variables.$primevue-prefix}-input-icon-right {
```
Comment (MCatherine1994) [inline style-sheets/primevue-components-overrides.scss:434]: If this class is for input, then let's place it under the input component section, not under the datable section

File: style-sheets/primevue-components-overrides.scss (hunk 3 above)
Comment (J0taFerreira) [reply style-sheets/primevue-components-overrides.scss:420]: This is for giving some space between the user name and the avatar icon.

File: style-sheets/primevue-components-overrides.scss (hunk 5 above)
Comment (J0taFerreira) [reply style-sheets/primevue-components-overrides.scss:434]: Will do, thanks!

File: style-sheets/primevue-components-overrides.scss (hunk 4 above)
Comment (J0taFerreira) [reply style-sheets/primevue-components-overrides.scss:424]: sure!

File: style-sheets/primevue-components-overrides.scss (hunk 5 above)
Comment (MCatherine1994) [reply style-sheets/primevue-components-overrides.scss:416]: Hi Jota, I saw you copied this to the input section, shall we remove it from here then?

Comment (J0taFerreira) [reply style-sheets/primevue-components-overrides.scss:416]: Hello! Agreed, I will do it ASAP!

**Review by MCatherine1994 — APPROVED** [review]

**Pull request merged**